lib/actions/ui.js

```diff
@@ -1,6 +1,6 @@
 import {php_escapeshellcmd as escapeShellCmd} from 'php-escape-shell';
-import last from '../utils/array';
-import isExecutable from '../utils/file';
+import {last} from '../utils/array';
```

Comment:
Only used here? (see comment further down)

Reply:
You mean that we have a module with only one function with only one line `arr[arr.length - 1]` imported in only one file and executed only one time: https://github.com/ShaggyDev/hyper/blob/d868721a0661b42d39be3eda086e0fd0d3342561/lib/actions/ui.js#L168

😂 

Ok, we need to remove this `utils/array.js` module